tests/testthat/test-dplyr.R

```diff
@@ -25,8 +25,11 @@ describe("pool package", {
     })
 
     it("can use dplyr syntax to copy table to DB", {
+
+      skip_if_not_installed("dplyr")
```

Comment:
No need for this skip; you can assume suggested packages will be available during testing.